src/components/file/examples/file-accepted-types.tsx

```diff
@@ -1,8 +1,7 @@
 import { FileInfo } from '@limetech/lime-elements';
 import { Component, h, State } from '@stencil/core';
-
 /**
- * Limit accepted file types
+ * Limit accepted file types and extentions
```

Comment:
We're not limiting the file extension, we're limiting the file type. It seems like at least macOS is deriving the file type from the file extension, so the effect on macOS is that the file extension is limited, but I'm not sure that's true for all systems.

Reply:
OK I'll revert back